### ISSUE TITLE
language package option can not be overridden from a specific language to auto

### ISSUE THREAD

**Issue by dcpurton** [plk/biblatex#903]
If a specific language is specified in the options, it does not seem to be possible to later revert to `language=auto` behaviour.

Consider this MWE:

```TeX
\documentclass{article}
\usepackage[english,ngerman]{babel}
\usepackage{csquotes}
\usepackage[style=verbose,language=english,autolang=other,language=auto]{biblatex}% always uses english
%\usepackage[style=verbose,autolang=other,language=auto]{biblatex}% works as expected
\addbibresource{biblatex-examples.bib}
\begin{document}
\null\vfill
Filler text \autocite{westfahl:space}.
Filler text \autocite{aksin}.
\printbibliography
\end{document}
```

**Comment by moewew** [comment]
Thanks for reporting this bug.

It should hopefully be fixed with https://github.com/plk/biblatex/commit/6fa64cd7e96f01ab39f7e593c63f83c553881b2c. Testing would be appreciated.

**Comment by dcpurton** [comment]
Seems to work with the example in https://tex.stackexchange.com/a/498826/87678 and does not break any of my tests with `biblatex-sbl`. Thanks!

**Comment by moewew** [comment]
Phew! Thanks for testing.

**Comment by moewew** [comment]
Should be fixed in version 3.13, which was released yesterday.
